session19/src/challenge2/LightColor.java

```diff
@@ -0,0 +1,44 @@
+package challenge2;
+
+public class LightColor {
+
+    enum Lights {
+        RED {
+            public String getRepresentation() {
+                return "Stop!";
+            }
+            public String getDuration() {
+                return "30 seconds";
+            }
+        },
+        YELLOW {
+            public String getRepresentation() {
+                return "Caution.";
+            }
+            public String getDuration() {
+                return "5 seconds";
+            }
+        },
+        GREEN {
+            public String getRepresentation() {
+                return "Go!";
+            }
+            public String getDuration() {
+                return "45 seconds";
+            }
+        };
+
+        public abstract String getRepresentation();
+        public abstract String getDuration();
```

Comment:
you can simplify these by making them simply fields on this enum and then create a constructor for this enum (not abstract methods) since they are always fixed values (no computation needed inside a method).